test/conformance/kuttl/cleanup/clusterpolicy/context-cleanup-pod/04-sleep.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: sleep 65
```

Comment:
Hmm 65s is a long delay, can this be improved? Does Kyverno create any event when it issues a DELETE request?

Reply:
It's set by cronJobs `schedule: "*/1 * * * *"` , The cronjob is scheduled to run every minutes, That's why sleeping time is `65s`, If CronJob scheduling time can be reduced then only sleeping time can be improved.